src/datepicker/test/datepicker.spec.js

```diff
@@ -1832,6 +1834,17 @@ describe('datepicker directive', function () {
       expect(element.find('tbody').find('tr').length).toBeGreaterThan(3);
     });
 
+    it('has next and previous icons', function () {
+
+      expect($rootScope.nextIconClass).toEqual('glyphicon glyphicon-chevron-right');
+      expect($rootScope.prevIconClass).toEqual('glyphicon glyphicon-chevron-left');
+
+      var nextIconClassSelector = $rootScope.nextIconClass.split(' ').map(function (c) { return '.' + c; }).join('');
+      var prevIconClassSelector = $rootScope.prevIconClass.split(' ').map(function (c) { return '.' + c; }).join('');
```

Comment:
The Rubyist in me likes this, but the every other part of me thinks this is gross.